tests/migration/migration.go

```diff
@@ -1017,7 +1017,36 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				Expect(err.Error()).To(ContainSubstring("DisksNotLiveMigratable"))
 			})
 			It("[test_id:1479][storage-req] should migrate a vmi with a shared block disk", decorators.StorageReq, func() {
-				vmi, _ := tests.NewRandomVirtualMachineInstanceWithBlockDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), k8sv1.ReadWriteMany)
+				if !libstorage.HasCDI() {
```

Comment:
@TheRealSibasishBehera you can refer to the [document](https://github.com/0xFelix/kubevirt/tree/coding-conventions/docs#short-coding-conventions-tldr) that Felix is writing as part of https://github.com/kubevirt/kubevirt/pull/11456 . It is still in progress but there is some guidelines to avoid `Skip` and use the decorators.

Hence you should assume that the feature and components are there and failing as consequence. Hence, this becomes:
```golang
Expect(libstorage.HasCDI()).To(BeTrue())
```

Reply:
yes ,i will make the changes . Thanks